makefiles/targets.mk

```diff
@@ -42,7 +42,7 @@ sanity-checks:
 warn-update:
 	if [[ -e $(UPDATE_TIMESTAMP) ]]; then \
 		if [[ $$(cat $(UPDATE_TIMESTAMP)) -ne $$($(GET_CURRENT_COMMIT_TIMESTAMP)) ]]; then \
-			echo -e "\n\"$$(basename "$(MAKE)") update\" was not executed for the current Git commit! The resulting build is unsupported.\n"; \
+			echo -e "\n\"$$(basename Warning: to ensure you are running Nimbus with a tested build, please always run "$(MAKE)") update\" after \"git pull\" (it looks like you've forgotten to do this). This also applies whenever you switch to a new branch or commit e.g. whenever you run \"git checkout\".\n"; \
```

Comment:
This project is used by more than Nimbus-eth2 and Nimbus-eth1: nim-waku, Stratus, nim-status-client, etc.

You also want to keep `\"$$(basename "$(MAKE)") update\"` intact, because it avoids showing a very long full path on Windows.

Reply:
i see... I do think it's not clear/specific enough as is.

if we need a clearer nimbus specific error message, how do would we go about it?

Reply:
We can inject the project name here with an environment variable specified in env.sh. It could default to the top-level repository name.

Reply:
Not worth complicating it that much. Further more, "running Nimbus with a tested build" doesn't really make sense. There is no testing here, just a supported state of the repository when all the submodules are up to date.

Reply:
This message should clearly tell you what is wrong and what to do next. Right now, it doesn't. It just causes panic.